schemas/glides-events.json

```diff
@@ -0,0 +1,112 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/mbta/schemas/blob/main/schemas/glides-events.json",
+  "title": "Glides Events",
+  "description": "Kinesis events emitted by Glides",
+  "$ref": "cloudevents.json",
+  "type": "object",
+  "properties": {
+    "type": {
+      "type": "string",
+      "description": "name of the event",
+      "$comment": "enforced to be a valid event name by the oneOf, below"
+    },
+    "specversion": { "const": "1.0" },
+    "source": { "const": "glides.mbta.com" },
```

Comment:
Note to self: This should probably be set differently per environment, instead of a const.